index.js

```diff
@@ -2274,6 +2274,24 @@
         return n < 0 || negativeZero(n) ? Nothing() : slice(0, -n, xs);
       });
 
+  //# append :: a -> [a] -> [a]
+  //.
+  //. Takes a value of any type and a list of values of that type, and
+  //. returns the result of appending the value to the list.
```

Comment:
In light of #180, we need to make a few changes to preserve consistency:

  - change the signature to `a -> Array a -> Array a`;
  - `s/a list/an array/`;
  - `s/the list/the array/`; and
  - move the function to the Array section (at the very beginning, let's say).

Much has changed in the few days since you submitted this pull request. I realize I've already asked you to move this function once! There won't usually be this much friction, I promise. :)